dashboard/app/views/levels/_external_link.html.haml

```diff
@@ -2,11 +2,17 @@
   - if @level.markdown_instructions
     %div= render(inline: @level.markdown_instructions, type: :md)
 
-  %h4= link_to @level.link_title, @level.url, target: '_blank'
+  %h4<
+    = link_to @level.url, target: '_blank' do
+      = @level.link_title
+      %i.fa.fa-external-link-square.external-link-icon>
 
-  %div{style: 'padding-top: 100px'}!= t('external_links.disclaimer_md', support_url: 'https://support.code.org')
+  .external-link-disclaimer
+    %h2= t('external_links.disclaimer')
+    %strong= t('external_links.leaving_domain')
+    %p!= t('external_links.disclaimer_text', support_url: 'https://support.code.org')
 
-  - if @script.try(:professional_learning_course?) && @script_level && @level.class == ExternalLink
+  - if @script_level
```

Comment:
The "Next resource" button should show up in all courses, not just PLC (although I need to update this to just say "Continue").